backend/apiServer/main.go

```diff
@@ -17,6 +18,7 @@ const (
 )
 
 func main() {
+	r := mux.NewRouter()
```

Comment:
rename to "router", more difficult to intuitively understand when it just named "r"